#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_skill_perror.dpatch by  <csmall@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: kill prints perror #468363

@DPATCH@
diff -urNad procps-3.2.7~/skill.c procps-3.2.7/skill.c
--- procps-3.2.7~/skill.c	2008-04-07 10:07:12.000000000 +1000
+++ procps-3.2.7/skill.c	2008-04-07 10:07:58.000000000 +1000
@@ -309,6 +309,7 @@
     pid = strtol(argv[argc],&endp,10);
     if(!*endp){
       if(!kill((pid_t)pid,signo)) continue;
+      perror("kill");
       exitvalue = 1;
       continue;
     }
